### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by valerix85**
Hi, I would like to contribute with the Italian translation ;)

### CODE REVIEW

**Comment by jeroenterheerdt** [comment]
Awesome, thank you! While we're in beta there might be more changes required to the language file. Also, please add one for the front end as well!

**Comment by gatonero** [comment]
For vscode I highly recommend using https://marketplace.visualstudio.com/items?itemName=Mohammed.json-translations-manager

**Comment by gatonero** [comment]
@valerix85 there are also language files in /custom_components/smart_irrigation/frontend/localize/languages

**Comment by valerix85** [comment]
> @valerix85 there are also language files in /custom_components/smart_irrigation/frontend/localize/languages

done

**Comment by gatonero** [comment]
@jeroenterheerdt I'm not used in merging. Some checks weren't successful. Could yoe please have a look. I don't want to break anything again.

**Comment by valerix85** [comment]
> Awesome, thank you! While we're in beta there might be more changes required to the language file. Also, please add one for the front end as well!

no problem, happy to help ;) if you want I can translate into other languages

**Comment by jeroenterheerdt** [comment]
> > Awesome, thank you! While we're in beta there might be more changes required to the language file. Also, please add one for the front end as well!
> 
> no problem, happy to help ;) if you want I can translate into other languages

Please double check with the current beta, I think the IT language files are behind already. I don't expect many changes any more though, so hopefully this is it.
Feel free to add whatever other language you speak, but maybe in separate PRs? I need to make small code changes per language supported so that it easier for me to track the work.

File: custom_components/smart_irrigation/frontend/localize/languages/it.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          },
+          "sensor-entity": "Entità sensore",
+          "sensor-units": "Il sensore fornisce valori in",
+          "sensor-aggregate-use-the": "Usa il",
+          "sensor-aggregate-of-sensor-values-to-calculate": "dei valori del sensore per calcolare la durata",
+          "aggregates": {
+            "average": "Media",
+            "minimum": "Minimo",
+            "maximum": "Massimo",
+            "first": "Prmo",
```
Comment (jeroenterheerdt) [inline custom_components/smart_irrigation/frontend/localize/languages/it.json:108]: Typo?

**Comment by jeroenterheerdt** [comment]
@valerix85 further updates: maximum bucket, localized help page and extra explanation line as well

File: custom_components/smart_irrigation/frontend/localize/languages/it.json (hunk 1 above)
Comment (valerix85) [reply custom_components/smart_irrigation/frontend/localize/languages/it.json:108]: doh! yes=( already fixed!

**Comment by valerix85** [comment]
> @valerix85 further updates: maximum bucket, localized help page and extra explanation line as well

updated all the it files

**Pull request merged**